test/util/login-helpers.js

```diff
@@ -52,16 +51,29 @@ function initUserMock(level) {
 }
 /* eslint-enable camelcase */
 
-function logIn(client, url, level) {
-  initUserMock(level);
+let tokenCounter = 0;
```

Comment:
none of the tests we're currently running require multiple signed in users. This little extra protection (as opposed to making the user unique to the pid) ensures that if we do find a way to run tests in parallel within a single process, this will still function as expected.